libpod/networking_common.go

```diff
@@ -632,6 +641,18 @@ func (r *Runtime) ConnectContainerToNetwork(nameOrID, netName string, netOpts ty
 	return ctr.NetworkConnect(nameOrID, netName, netOpts)
 }
 
+// hostInterfaceName takes a network name, a partial or a full network ID and
+// returns the host interface name.
+// return an empty string in case of any error.
+func (r *Runtime) hostInterfaceName(nameOrID string) (string, error) {
```

Comment:
this function does not compile

also please do not inspect the same network twice, just change normalizeNetworkName() to also return you the interface name.